hack/scale_out_poc/config_haproxy/setup_haproxy.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
```

Comment:
An updated version of this script has already been moved to master branch in https://github.com/CentaurusInfra/arktos/pull/929. This file should be removed.

Reply:
deleted